rust/tests/behaviour/query/reasoner/steps.rs

```diff
@@ -103,6 +103,6 @@ generic_step_impl! {
 
     #[step(expr = "verify answers are consistent across {int} executions")]
     async fn verify_answers_are_consistent_across_executions(_context: &mut Context, _executions: usize) {
-        //     We can't execute previous query again because don't remember the query
+        //     We can't execute previous query again because don't remember the query // TODO: start saving the query and implement
```

Comment:
Maybe we want to create a task AND reference this task from the code so these TODOs could be found more easily?

Reply:
#612

Reply:
Cool, thank you! What do you think about starting referencing github issues inside the code TODOs? For example:
` // TODO #612: start saving the query and implement`

It would help us cleaning outdated todos and remembering reasons behind each of them on a long distance.